logs/.gitignore

```diff
@@ -0,0 +1 @@
+!.gitignore
```

Comment:
You should ignore log files here, e.g., `*_log.php` or just `*`

Reply:
> You should ignore log files here, e.g., `*_log.php` or just `*`

`!.gitignore` can not work?

Reply:
Okey

Reply:
```
*
!.gitignore
```

is okay, however if you just use `!.gitignore` it won't work